go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	github.com/spaolacci/murmur3 v1.1.0 // indirect
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/syndtr/goleveldb v1.0.1-0.20190318030020-c3a204f8e965 // indirect
+	github.com/syndtr/goleveldb v1.0.1-0.20190318030020-c3a204f8e965
```

Comment:
this is not correct? you can run `go mod tidy` again.

Is there some un-committed code in your local repo still using it?

Reply:
yes， already revert this change